components/webext-storage/src/sync/bridge.rs

```diff
@@ -38,56 +46,74 @@ impl<'a> BridgedEngine<'a> {
         delete_meta(tx, LAST_SYNC_META_KEY)?;
         Ok(())
     }
-}
 
-impl<'a> sync15::engine::BridgedEngine for BridgedEngine<'a> {
-    type Error = Error;
+    fn get_shared_db(&self) -> Result<Arc<ThreadSafeStorageDb>> {
```

Comment:
nitpick: Let's name this method `thread_safe_storage_db()`, to match the new name you gave the type.